docs/examples/git-repo/kustomization-comp.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.oam.dev/v1beta1
+kind: ComponentDefinition
+metadata:
+  name: kustomize-control-repo
```

Comment:
`GitRepo` seems to be a better name.

Reply:
I don’t know whether my thought is right, from my point of view, the gitrepo-kind yaml describes the raw k8s resource about the git repository, Maybe the GitRepo name is better for it. While the component function is enhanced by kustomize controller.

Reply:
oh, I think I probably understand what you mean. if we use CUE to glue them together, we can call it a enhanced GitRepo ?

Reply:
Yes. `GitRepo` is a better name after glued together, `Kustomize` is also fine to me (though it can't work with local repo).